benchmark/benchmark.go

```diff
@@ -75,18 +81,22 @@ func (s *testServer) UnaryCall(ctx context.Context, in *testpb.SimpleRequest) (*
 }
 
 func (s *testServer) StreamingCall(stream testpb.BenchmarkService_StreamingCallServer) error {
+	response := new(testpb.SimpleResponse)
```

Comment:
Can we just assign payload to the response object here so that we won't have to do the assignment inside the loop.